gc/base/MemoryPoolAddressOrderedList.hpp

```diff
@@ -98,6 +102,10 @@ class MM_MemoryPoolAddressOrderedList : public MM_MemoryPoolAddressOrderedListBa
 	 */
 	MM_HeapLinkedFreeHeader *doFreeEntryAlignmentUpTo(MM_EnvironmentBase *env, MM_HeapLinkedFreeHeader *lastFreeEntryToAlign);
 
+	MMINLINE bool isAlignementForParallelGCRequired() {
+		return (NULL != _tlhAlignmentBase);
+	}
```

Comment:
to avoid confusion between 2 different types of alignement:
- move this next to alignTLHForParallelGC
- all methods and fields for the existing alignement append with a word 'Card': doesNeedCardAlignment, _firstCardUnalignedFreeEntry,....

Reply:
_tlhAlignmentBase and _tlhAlignmentSize are neither specific enough (with its 'tlh'), although I don't have a good suggestion... something like parallelGCAlignmentBase/Size?